tests/.infra/crw-ci/single-host/k8s/Jenkinsfile

```diff
@@ -0,0 +1,176 @@
+pipeline {
+    agent { label 'rhel7-8gb' }
+
+    options {
+        timestamps()
+        timeout(time: 6, unit: 'HOURS')
+        buildDiscarder(logRotator(artifactDaysToKeepStr: '',
+                artifactNumToKeepStr: '', daysToKeepStr: '60', numToKeepStr: '30'))
+    }
+
+    triggers { cron("H 1 * * *") }
+
+    parameters {
+        string(name: 'emails',
+                defaultValue: 'oshmarai@redhat.com',
+                description: 'Comma-separated list of emails to send results')
+    }
+
+    stages {
+        stage("Run E2E tests") {
+            steps {
+                script {
+                    catchError(buildResult: 'UNSTABLE', stageResult: 'FAILED') {
+                        stage('Run Happy path tests') {
+                            retry(2) {
+                                build job: 'basic-MultiUser-Che-check-e2e-tests-against-k8s',
+                                        parameters: [
+                                                booleanParam(name: 'cheSingleHost',
```

Comment:
Parameter `cheSingleHost` should be added to **tests/.infra/crw-ci/nightly/k8s/Jenkinsfile** pipeline as well, to make it possible to call updated **tests/.infra/crw-ci/pr-check/k8s/Jenkinsfile** pipeline.

Reply:
Good thought. Done.